floyd/cli/experiment.py

```diff
@@ -182,12 +182,33 @@ def get_log_id(job_id):
     return instance_log_id
 
 
+def check_job_termination_line(log_output_line):
+    """
+    Get Job termination by inspecting if any of the termination log lines is detected
+    in the current log output line.
+    """
+    job_terminated = False
+    success = "[success] Finished execution"
+    failure = "[failed] Task execution failed"
+    shutdown = "[shutdown] Task execution cancelled"
+    timeout = "[timeout] Task execution cancelled"
+    termination_list = [success, failure, shutdown, timeout]
```

Comment:
better move this list creation outside so we are not creating a new list on every function call.

Reply:
Agreed, move this outside the function to the top of this file. Also since these are constants use capitalized variable names for easy identification.

```
SUCCESS_OUTPUT = "[success] Finished execution"
FAILURE_OUTPUT = "[failed] Task execution failed"
..
TERMINATION_OUTPUT_LIST = [SUCCESS_OUTPUT, FAILURE_OUTPUT, ..]
```